### DISCUSSION TITLE
MAUI create a pdf same as content page

### DISCUSSION THREAD

**Discussion by vsfeedback** [dotnet/maui#24202] [Q&A]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/MAUI-create-a-pdf-same-as-content-page/10710831)._

---
Greetings,
              I m recently working on a project which required make pdf from Content pages. may anyone suggest what I can do to achieve it. Firstly I uses itext7 for it but it won't add image to pdf only text is appearing in pdf, may anyone know solution.

**Comment by jarlem** [comment]
I suggest to have that page displayed as Blazor (hybrid) because HTML is easy to generate PDFs from. The generation of the PDF would be handled server side. Just pass the HTML content to the server and get a PDF in return.
